labgrid/driver/rawnetworkinterfacedriver.py

```diff
@@ -0,0 +1,97 @@
+# pylint: disable=no-member
+import json
+import os
+import signal
+import subprocess
+
+import attr
+
+from .common import Driver
+from ..factory import target_factory
+from ..step import step
+from ..util import sshmanager
+from ..util.helper import processwrapper
+from ..util.managedfile import ManagedFile
+from ..resource.remote import RemoteNetworkInterface
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class RawNetworkInterfaceDriver(Driver):
+    bindings = {
+        "iface": {"NetworkInterface", "RemoteNetworkInterface", "USBNetworkInterface"},
+    }
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self.proc_pcap_map = {}
+
+    def _get_wrapper_prefix(self):
+        return self.iface.command_prefix + ["sudo", "labgrid-raw-interface"]
+
+    def _remote_tmp_file_name(self):
+        cmd = self.iface.command_prefix + ["mktemp", "--dry-run"]
```

Comment:
We should use `mktemp -d` here to avoid potential races and security issues. In the created directory, only the user has write access.

Reply:
Ok, agreed.

Reply:
This is not needed anymore, we use stdout and capture into a NamedTemporaryFile instead.